en/docs/how-tos/analytics-for-microgateway.md

```diff
@@ -203,10 +215,10 @@ The configurations are described in the table below.
 
```

Comment:
Lets also mention here these steps are required only if real time publishing is used. And methiion we have to configure only one of them not the both

Reply:
Done with https://github.com/wso2/docs-mg/pull/124/commits/d6479400b0cf6238740347e88d41211234f72980